### PR TITLE
[zalando-postgres-cluster ] Enable additional settings for dumpBackup

### PR DESCRIPTION
**Pull request by angelnu**
**Description of the change**

Allow the dumpBackup to take options of the common lib-chart persistence volume.

**Benefits**

Additional settings for the backup volume

**Possible drawbacks**

None: only minor update so compatible with old values.

**Applicable issues**

- fixes NA

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/zalando-postgres-cluster/values.yaml [hunk 1]
```diff
@@ -59,8 +59,12 @@ persistentVolumes:
 
 
 dumpBackup:
-  # Enable backups to a PVC
+  # -- Enable backups to a PVC
+  enabled: false
+  # -- exiting claim
```
Comment (bjw-s) [inline charts/stable/zalando-postgres-cluster/values.yaml:64]:
```suggestion
  # -- existing claim
```

File: charts/stable/zalando-postgres-cluster/values.yaml [hunk 2]
```diff
@@ -59,8 +59,12 @@ persistentVolumes:
...
   existingClaim:
+  # -- type of persistence volume to use
```
Comment (bjw-s) [inline charts/stable/zalando-postgres-cluster/values.yaml:66]: Maybe add some guidance about the options here, or refer to the storage docs?

File: charts/stable/zalando-postgres-cluster/templates/backupCronjob.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.dumpBackup.existingClaim -}}
+{{- if or .Values.dumpBackup.enabled .Values.dumpBackup.existingClaim -}}
```
Comment (bjw-s) [inline charts/stable/zalando-postgres-cluster/templates/backupCronjob.yaml:1]: Did you put the `or` in here to make it a non-breaking change?

Comment (angelnu) [reply charts/stable/zalando-postgres-cluster/templates/backupCronjob.yaml:1]: exactly, it is to make it non-breaking

File: charts/stable/zalando-postgres-cluster/values.yaml (hunk 2 above)
Comment (angelnu) [reply charts/stable/zalando-postgres-cluster/values.yaml]: Added

**Review by bjw-s — APPROVED** [review]

**Pull request merged**